### ISSUE TITLE
Add Doxygen To Webpage

### ISSUE THREAD

**Issue by danscime** [JacquesCarette/Drasil#1763]
@bmaclach recently added Doxygen to Drasil. Generated Doxygen pages are definitely something to add to ur webpage.

**Comment by smiths** [comment]
Sounds good!

**Comment by bmaclach** [comment]
@smiths Should we turn Doxygen comments on for all of the examples for which we generate code so that we can link to Doxygen for each of them in the webpage?

Currently Doxygen comments are fully enabled for GlassBR, enabled only for functions for Projectile, and disabled for NoPCM.

**Comment by smiths** [comment]
Yes, @bmaclach, I like the idea of turning on doxygen comments for all of the examples.  Without too much more work, we'll make all of the examples more impressive.  😄

**Comment by danscime** [comment]
We need a way to create generate Doxygen page and load them into the `gh-pages` branch in order to be used by the site.
